src/net.cpp

```diff
@@ -1768,11 +1768,11 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
     }
 
     // Initiate network connections
-    int64_t nStart = GetTime();
+    auto start = GetTime<std::chrono::microseconds>();
```

Comment:
I think `GetTime` should actually return a `std::chrono::time_point` instead of a duration. But that's probably out of scope for this PR. Then we'd have a few more compile time checks concerning the allowed operations (e.g. you can't add two timepoints, you can't accidentally compare a timepoints with a duration and you can't accidentally mix timepoints from different clocks)